site/composer.json

```diff
@@ -34,7 +34,7 @@
 		"exercise/htmlpurifier-bundle" : "*",
 		"doctrine/doctrine-migrations-bundle" : "^1.0",
 		"symfony/http-kernel" : "~3.2.3",
-		"knplabs/github-api": "^2.3",
+		"knplabs/github-api": "dev-master",
```

Comment:
Does it mean "latest"? If yes, it may cause instability when a new release comes out